rules-tests/Php80/Rector/Class_/AnnotationToAttributeRector/config/configured_rule.php

```diff
@@ -17,6 +17,7 @@
 
                 new AnnotationToAttribute('Symfony\Component\Routing\Annotation\Route'),
                 new AnnotationToAttribute('Symfony\Component\Validator\Constraints\Choice'),
+                new AnnotationToAttribute('Sensio\Bundle\FrameworkExtraBundle\Configuration\Security'),
```

Comment:
This is not neccesery to test attribute inner syntax. Could re-use one of already added annotations, e.g. Route?